codeforamerica/vita-min
# FYST-952 Update interest on government bonds field to pull from 1099-INT accessor class

## Link to pivotal/JIRA issue
- https://codeforamerica.atlassian.net/browse/FYST-952
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
We removed the fake direct file field interest_reported_amount and replaced it with the sum of the interest reports' interest_on_government_bonds.
## How to test?
Use a 1099 df that includes interest reports to see if the data appears in the XML and PDF.
